.autotest/agent-test.sh

```diff
@@ -62,7 +62,7 @@ checkoutSourceCode(){
 		git clone $REPO_URL "$SOURCE_CODE_DIR" 1>&2 > /dev/null
 	fi
 
-	eval "cd $SOURCE_CODE_DIR && git reset --hard && git checkout $REPO_BRANCH && git fetch origin && git pull origin $REPO_BRANCH" 1>&2 > /dev/null
+	eval "cd $SOURCE_CODE_DIR && git reset --hard && git remote set-branches origin $REPO_BRANCH && git fetch --depth 1 origin $REPO_BRANCH && git checkout $REPO_BRANCH && git fetch origin && git pull origin $REPO_BRANCH" 1>&2 > /dev/null
```

Comment:
Why did you edit this file?

Reply:
Shallow clone will be faster then full clone. Full clone will take a long time to test repeatedly, it's very slow.

Reply:
If that's okay, need another PR ? :)

Reply:
That' OK, You can pull another PR to fix this.

Reply:
I think that `Shallow clone` can also send another PR

Reply:
I think you miss `git checkout to the $REPO_BRANCH`

Reply:
I have reverted the modify for 'agent-test.sh' and two PR are sent into branch 'feature/5.x': #30 #31